src/NLog.StructuredLogging.Json/LoggerExtensions.cs

```diff
@@ -119,10 +120,28 @@ private static void TransferDataToLogEventProperties(LogEventInfo log, object lo
             }
         }
 
+        private static void TransferContextDataToLogEventProperties(LogEventInfo log)
+        {
+            foreach (var contextItemName in MappedDiagnosticsLogicalContext.GetNames())
+            {
+                var value = MappedDiagnosticsLogicalContext.Get(contextItemName);
```

Comment:
Can do, though we expect that the case that we don't use it, because the first choice name is already in use, and so is the second choice name is a bit of an unusual edge case.